mudirac/mudirac.xml

```diff
@@ -198,18 +198,18 @@ spec_expdec: $spec_expdec
         </conditional>
     </inputs>
     <outputs>
-        <data name="output" format="tabular" label="Output of ${tool.name}" from_work_dir="input.xr.out"/>
+        <data name="output" format="tabular" label="Output of ${tool.name} (${element})" from_work_dir="input.xr.out"/>
         <collection name="states" format="tabular" label="States" type="list">
```

Comment:
Is there a reason to not include the element in the label for this output? As it's present in all the others should we be consistent?

```suggestion
        <collection name="states" format="tabular" label="States (${element})" type="list">
```